src/Umbraco.Web/Trees/ITreeNodeController.cs

```diff
@@ -3,6 +3,7 @@
 using Umbraco.Web.WebApi.Filters;
 using System.Web.Http.ModelBinding;
 
+//Migrated to .NET CORE
```

Comment:
Are these temporary comments for future? Just to make sure they don't get left in and forgotten if temp

Reply:
In the future these classes with be removed. But currently, these are used by other classes or tests, that is not migrated yet.

Eg.

`ContentControllerTests`, uses the content mapper, which uses this interface. So this class can be removed as soon as all references are migrated.